core/src/smartcontracts/isi/account.rs

```diff
@@ -261,13 +261,6 @@ pub mod isi {
             // Check if account exists
             wsv.account_mut(&account_id)?;
 
-            let definition = wsv
-                .permission_token_definitions()
-                .get(&permission_id)
-                .ok_or_else(|| FindError::PermissionTokenDefinition(permission_id.clone()))?;
```

Comment:
this only makes sense to be done in the validator itself. Since Iroha doesn't use the concept of `PermissionToken` in any way anymore. Registering a wrong definition has no implications for `Iroha`